Code/GraphMol/FMCS/FMCS.h

```diff
@@ -37,13 +37,31 @@ typedef enum {
   StrictRingFusion
 } RingComparator;
 
+typedef std::map<const Atom*, std::map<const Atom*, bool>> MCSAtomDistanceCache;
+namespace FMCS {
+class RingMatchTableSet;
+}
+struct RDKIT_FMCS_EXPORT MCSCompareFunctionsData {
```

Comment:
This is the struct that gets passed around to the compare functors now. It can hold a genuine userData pointer,  but it doesn't do anything with it!